device-types/Cisco/CBS250-8PP-D.yml

```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: Cisco
+model: CBS250-8PP-D
+slug: cisco-cbs250-8pp-d
+part_number: CBS250-8PP-D
+comments: '[CBS250 series datasheet](https://www.cisco.com/c/en/us/products/collateral/switches/business-250-series-smart-switches/nb-06-bus250-smart-switch-ds-cte-en.html)'
+airflow: passive
+weight: 0.56
+weight_unit: kg
+is_full_depth: false
+front_image: true
+u_height: 1
+interfaces:
+  - name: GigabitEthernet1
+    type: 1000base-t
+    poe_mode: pd
+    poe_type: type1-ieee802.3af
```

Comment:
Looking at the spec sheet these are POE+ therefore they should be `poe_type: type2-ieee802.3at`